rbs_collection.yaml

```diff
@@ -0,0 +1,28 @@
+sources:
+  - type: git
+    name: ruby/gem_rbs_collection
+    remote: https://github.com/ruby/gem_rbs_collection.git
+    revision: main
+    repo_dir: gems
+
+path: .gem_rbs_collection
+
+gems:
+  - name: addressable
```

Comment:
I'm not sure if theres preferred ways to organize this.  It might be easier to read if all of the included stdlib (tempfile/stringio) are listed first, then all of the ignored ones?

Reply:
Yeah I think that might be better - all the ignored ones will be alphabetical